cmd/ingester/app/flags_test.go

```diff
@@ -49,6 +53,45 @@ func TestOptionsWithFlags(t *testing.T) {
 	assert.Equal(t, kafka.EncodingJSON, o.Encoding)
 }
 
+func TestTLSFlags(t *testing.T) {
+	tests := []struct {
+		flags    []string
+		expected auth.AuthenticationConfig
+	}{
+		{
+			flags:    []string{},
+			expected: auth.AuthenticationConfig{Authentication: "none", Kerberos: auth.KerberosConfig{ServiceName: "kafka", ConfigPath: "/etc/krb5.conf", KeyTabPath: "/etc/security/kafka.keytab"}},
```

Comment:
Why do we have Kerberos populated in all these tests? It looks unexpected.

If it must be there, let's pull that value into a constant, to make the tests easier to read. Or even set the expected value inside the test itself, to not obscure the meaning.